index.js

```diff
@@ -201,3 +201,46 @@ modal.addEventListener('click', (e) => {
 });
 
 generateProduct();
+
+const button = document.querySelector('#form_button');
+
+button.addEventListener('click', (event) => {
+  event.preventDefault();
+  // form validation
+  const contactForm = document.getElementById('contactForm');
+  const fullName = document.querySelector('#name');
+  const email = document.querySelector('#email');
+  const textarea = document.querySelector('#textarea');
+  const messageErr = document.querySelector('#message');
+  const checkLower = email.value;
+
+  if (fullName.value === '') {
+    messageErr.innerHTML = 'Please provide your name';
+    fullName.focus();
+    // return false;
+  } else if (email.value === '') {
+    messageErr.innerHTML = 'Please provide email';
+    email.focus();
+    // return false;
+  } else if (checkLower !== checkLower.toLowerCase()) {
+    messageErr.innerHTML = 'Please use lowercase for email';
+    email.focus();
```

Comment:
- [OPTIONAL] Kindly consider improving your error message color, Il will be much better if you could use a strong color for the text error message. 👍🏻